src/plugins/intel_npu/src/plugin/npuw/compiled_model.cpp

```diff
@@ -526,6 +530,49 @@ ov::npuw::CompiledModel::CompiledModel(const std::shared_ptr<ov::Model>& model,
     LOG_DEBUG("CompiledModel is being deserialized, skipping the full constructor flow...");
 }
 
+void ov::npuw::CompiledModel::identify_host_gather_property(const std::shared_ptr<ov::Model>& model) {
+    // Identify based on the config
+    try {
+        auto compiler_version = get_plugin()->get_core()->get_property("NPU", ov::intel_npu::compiler_version);
+        if (compiler_version >= ONEAPI_MAKE_VERSION(7, 21)) {
+            // Force quantized host gather
+            LOG_INFO("Forcing NPUW_HOST_GATHER_QUANT property.");
+            std::map<std::string, std::string> host_gather_cfg;
+            host_gather_cfg["NPUW_HOST_GATHER_QUANT"] = "YES";
+            host_gather_cfg["NPUW_HOST_GATHER"] = "NO";
```

Comment:
so.. would that essentially cancel the HOST_GATHER features for models where HOST_GATHER_QUANT isn't requried?

Reply:
I don't think this is the right point for the decision making - you have one in the code below.
All you need to find here is if is is the right compiler version or not. so I'd just store it like `supports_asym_vocab = true` and that's it.